demo/desktop/vmf-player/MetadataProvider/MetadataProvider.cpp

```diff
@@ -301,10 +303,10 @@ void MetadataProvider::execute()
             vmf::Format::AttribMap attribs;
             vmf::Format::ParseCounters c;
 
-            char buf[10240];
-
             if (!m_exiting)
             {
+                char buf[bufSize];
```

Comment:
Perhaps it would be better to change the buffer to be the class member - 
it would be allocated/deallocated once as part of ctr/dtr logic.

On 04/22/2016 11:19 AM, Andrey Pavlenko wrote:
>
> In demo/desktop/vmf-player/MetadataProvider/MetadataProvider.cpp 
> <https://github.com/apavlenko/vmf/pull/64#discussion_r60705238>:
>
> >              if (!m_exiting)
> >              {
> > +                char buf[bufSize];
>
> agree, let's leave it as is
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/apavlenko/vmf/pull/64/files/bdf5cf8cc023f114d68b3eb0f69e027f9f821655..aa5c5d0514964726321eef42169f6865292a280a#r60705238>
>